core/templates/dev/head/css/oppia.css

```diff
@@ -95,19 +95,32 @@ md-input-group.md-default-theme label {
 
 .md-button.oppia-learner-confirm-button,
 .md-button.oppia-learner-got-it-button,
-.md-button.oppia-learner-try-again-button,
 .md-button.oppia-learner-hint-solution-button {
   background: #0D48A1;
   color: #ffffff;
   padding: 6px 12px;
 }
 
+.md-button.oppia-learner-try-again-button
```

Comment:
Lint: here and below, follow the style elsewhere please. Look at the other rules in this file and see how they are formatted.

Reply:
Done